test/twitter/accounts_test.exs

```diff
@@ -0,0 +1,78 @@
+defmodule Twitter.AccountsTest do
+  use Twitter.DataCase, async: true
+
+  alias Twitter.Accounts
+  alias Twitter.Accounts.User
+
+  describe "create_user/1" do
+    @valid_attrs %{
+      name: "User",
+      username: "eva",
+      password: "secret",
+      email: "blabla"
+    }
+    @invalid_attrs %{}
+
+    test "with valid data inserts user" do
+      assert {:ok, %User{id: id} = user} = Accounts.create_user(@valid_attrs)
+      assert user.name == "User"
+      assert user.username == "eva"
+      assert user.email == "blabla"
+      assert [%User{id: ^id}] = Accounts.list_users()
```

Comment:
make this one line with pattern match. combine everything from list_users. No need to check the returned `user.name, user.username user.email` from `Accounts.create_user`. This is really just implementation detail. I think you can trust the Repo.insert will return the correct thing for you. No need to test.

You should however pattern match on list_users in one line if you want to check (you could do a get_user_by instead of list_users as well if you wanted). Under current implementation, if you added seed data or something this test would fail making it somewhat brittle.

Reply:
Ok, fixed that